app/contact/mail.json

```diff
@@ -0,0 +1 @@
6,-411],[416,26],[-74,309],[-430,-32]],"c":true}]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":150,"s":[{"i":[[-263.72,24.16],[16.66,-322.3],[194.54,24.06],[76.48,256.34]],"o":[[263.72,-24.16],[-16.66,322.3],[-198.486,-24.548],[-76.48,-256.34]],"v":[[-47.72,-384.84],[380.66,62.7],[-56.54,346.94],[-379.52,15.34]],"c":true}]},{"t":199,"s":[{"i":[[-194.546,54.872],[46,-283],[202.136,0],[53.627,210.319]],"o":[[234,-66],[-34.823,214.236],[-202.136,0],[-64,-251]],"v":[[-58,-395],[382,-4],[-28,403],[-382,2]],"c":true}]}],"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"fl","c":{"a":0,"k":[0.47843137254901963,0.7098039215686275,0.11764705882352941,1],"ix":4},"o":{"a":0,"k":20,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[2,-19],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Ellipse 3","np":2,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false},{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":1,"k":[{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":0,"s":[{"i":[[-194.546,54.872],[46,-283],[202.136,0],[53.627,210.319]],"o":[[234,-66],[-34.823,214.236],[-202.136,0],[-64,-251]],"v":[[-58,-395],[382,-4],[-28,403],[-382,2]],"c":true}]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":50,"s":[{"i":[[-201.9,9.769],[46,-283],[198.441,-38.473],[-30.055,214.957]],"o":[[248,-12],[-34.823,214.236],[-196,38],[46,-329]],"v":[[-86,-411],[416,26],[-28,369],[-384,26]],"c":true}]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":99,"s":[{"i":[[-189.571,70.156],[-42,-269],[198.441,-38.473],[0,217.048]],"o":[[254,-94],[50.213,321.602],[-196,38],[0,-265]],"v":[[-86,-411],[416,26],[-74,309],[-430,-32]],"c":true}]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":150,"s":[{"i":[[-263.72,24.16],[16.66,-322.3],[194.54,24.06],[76.48,256.34]],"o":[[263.72,-24.16],[-16.66,322.3],[-198.486,-24.548],[-76.48,-256.34]],"v":[[-47.72,-384.84],[380.66,62.7],[-56.54,346.94],[-379.52,15.34]],"c":true}]},{"t":199,"s":[{"i":[[-194.546,54.872],[46,-283],[202.136,0],[53.627,210.319]],"o":[[234,-66],[-34.823,214.236],[-202.136,0],[-64,-251]],"v":[[-58,-395],[382,-4],[-28,403],[-382,2]],"c":true}]}],"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"fl","c":{"a":0,"k":[0.47843137254901963,0.7098039215686275,0.11764705882352941,1],"ix":4},"o":{"a":0,"k":15,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[2,-75],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[112,112],"ix":3},"r":{"a":0,"k":-174,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Ellipse 2","np":2,"cix":2,"bm":0,"ix":2,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":200,"st":0,"bm":0}],"markers":[]}
```

Comment:
It's a `Lottie asset` and it's a bit confusing that it's in the `contact` directory, IMO one thing you can do is create a new `assets` folder and move it there. So that it's easy to understand what is it's for.